<launch>
    <node pkg="tf" type="static_transform_publisher" name="base_to_link" args="0 0 0 0 0 0 base_footprint base_link 100"/>
    <node pkg="tf" type="static_transform_publisher" name="base_to_camera" args="0 0 0 0 0 0 base_footprint camera_link 100" />
    <node pkg="robot_start" type="lab306_robot_start" name="publish_odom" output="screen">
        <param name="robot_frame_id" type="string" value="base_link"/>
        <param name="smoother_cmd_vel" type="string" value="/my_cmd_vel"/>
        <param name="PubPath" type="bool" value="true"/>
        <param name="PubOdom" type="bool" value="true"/>
        <param name="PubMarkers" type="bool" value="true"/>
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find robot_start)/rviz/robot_start.rviz" required="true"/>

    <include file="$(find yocs_velocity_smoother)/launch/standalone.launch" >
    </include>
    <include file="$(find robot_start_lkw)/launch/display.launch">
    </include>
    <include file="$(find ORB_SLAM2)/launch/orbslam.launch"></include>
    <include file="$(find robot_pose_ekf)/robot_pose_ekf.launch"/>
</launch>
